### ISSUE TITLE
TASK: building ELN with dnf5? 

### ISSUE THREAD

**Issue by praiskup** [fedora-eln/eln#175]
### What does the ELN SIG need to do?
* decide when to make this change and coordinate the switch

We should follow similar steps as with Fedora Rawhide:
https://fedoraproject.org/wiki/Changes/BuildWithDNF5

Mock/Copr change:
https://github.com/rpm-software-management/mock/pull/1291

**Comment by praiskup** [comment]
Gently ping, what is the plan here?

**Comment by praiskup** [comment]
Gently ping again.  Any plans?

**Comment by sgallagher** [comment]
Thanks for the ping. I'm taking a look at https://github.com/rpm-software-management/mock/issues/1292 right now.

**Comment by yselkowitz** [comment]
Wouldn't the ELN container image (used in bootstrapping) need to have dnf5 first?

**Comment by sgallagher** [comment]
That would make it much faster, but it's not a strict requirement. Regardless, I'm already looking at updating the bootstrap image as well.

**Comment by sgallagher** [comment]
I have patches out for review for [Content Resolver](https://github.com/minimization/content-resolver-input/pull/1105) and [fedora-kickstarts](https://pagure.io/fedora-kickstarts/pull-request/1025). Once those land, we can update the [mock PR](https://github.com/rpm-software-management/mock/pull/1291) to not disable the `bootstrap_ready` option.

Also, I've done some local testing with `dnf5` as the package manager in mock for ELN and it worked just fine.

**Comment by sgallagher** [comment]
The two MRs above have been merged. Once the next compose runs in a couple hours, we should be able to use the new container image as a bootstrap image for mock.

**Comment by praiskup** [comment]
Is it the right time to make the change in Mock?

**Comment by yselkowitz** [comment]
Any bugs building packages with dnf5 should already be fixed, so I have no further objections to using dnf5 in mock and in koji for ELN.  @sgallagher does anything else need to be done first?

**Comment by sgallagher** [comment]
Let's do it.

**Comment by yselkowitz** [comment]
Filed https://pagure.io/releng/issue/12103 for koji
Pinged on https://github.com/rpm-software-management/mock/pull/1291

**Comment by yselkowitz** [comment]
The mock PR was merged and the koji tag modified; anything else?

**Comment by praiskup** [comment]
No, this seems done, thank you all! (I mean, we still have to wrap a new mock-core-configs release, and deploy that into Fedora Copr, but that's going to happen very soon)
